packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.integration.test.tsx

```diff
@@ -0,0 +1,183 @@
+import React from 'react'
```

Comment:
Last PR accidentally deleted this test suite. It was moved into `src/` to be type checked and converted into an integration test that uses a real `QueryWrapper`. The tests are otherwise mostly unchanged.